htdocs/assets/js/util.js

```diff
@@ -4,15 +4,27 @@ define(function(require) {
         _ = require('underscore'),
         URI = require('uri'),
         Autosize = require('autosize'),
-        CodeMirror = require('codemirror');
+        CodeMirror = require('codemirror'),
+        Moment = require('moment'),
+        Data = require('data');
+        // loads tz data
+        require('moment-timezone');
```

Comment:
I'd have to load both moment and moment-timezone there because without loading the core moment library first there isn't anywhere for moment-timezone to load the tz data into. I'll see what I can do!